packages/frontend-shared/src/locales/en-US.json

```diff
@@ -322,9 +322,9 @@
           "text": "Configuration guide",
           "link": "https://docs.cypress.io/guides/references/configuration"
         },
-        "dashboardGuide": {
-          "text": "Cypress Dashboard guide",
-          "link": "https://docs.cypress.io/guides/dashboard/introduction"
+        "cloudGuide": {
+          "text": "Cypress Cloud guide",
+          "link": "https://on.cypress.io/features-dashboard"
```

Comment:
I changed this to use an on link because the dashboard link will be invalid once the docs PR is merged

Reply:
Good call, I noticed the other day that all links from the app are supposed to be onlinks, including links to docs, so we probably have a little audit on our hands: https://github.com/cypress-io/cypress/blob/develop/CONTRIBUTING.md#adding-links-within-code